packages/-ember-data/tests/integration/identifiers/configuration-test.ts

```diff
@@ -254,9 +277,19 @@ module('Integration | Identifiers - configuration', function (hooks) {
     let updateMethodCalls = 0 as number;
     let updateCallback: (...args: any[]) => void;
 
-    function updateMethod(identifier: StableRecordIdentifier, data: ExistingResourceObject, bucket: string) {
-      updateMethodCalls++;
-      updateCallback!(identifier, data);
+    function updateMethod(
+      identifier: StableIdentifier | StableRecordIdentifier,
```

Comment:
Isn't a StableIdentifier a subset of StableRecordIdentifier, you shouldn't need both?

Reply:
It is but once you activate strict type signatures you can't implement something that accepts only the wider interface.